aemedge/templates/blog-article/blog-article.js

```diff
@@ -142,6 +142,7 @@ export async function buildAuthorBlock() {
   socialContainer.append(shareText, socialList);
   authTxtContainer.append(socialContainer);
   const section = document.createElement('div');
+  section.classList.add('rtl');
```

Comment:
You can remove this, I assume it is leftover from your previous work on the Arabic ticket....